package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "book",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.4",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "5.0.0",
+    "web-vitals": "^2.1.4"
+  },
```

Comment:
- [ ] react-router-dom should be present in package.json. It will be needed to make the routes. 😉

Reply:
Thank you for the positive feedback @eri8-9 👍